### ISSUE TITLE
Failing test(s): TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample

### ISSUE THREAD

**Issue by SarahFrench** [hashicorp/terraform-provider-google#20008]
### Impacted tests

- TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample
- TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMinAndMaxScalingFactorExample
- TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMinScalingFactorExample

### Affected Resource(s)

- google_dataproc_metastore_service

### Failure rates

- 100% since 2024-07-22 in GA

### Message(s)

```
------- Stdout: -------
=== RUN   TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample
=== PAUSE TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample
=== CONT  TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample
    resource_dataproc_metastore_service_generated_test.go:591: Step 1/2 error: Error running pre-apply refresh: exit status 1
        Error: Unsupported block type
          on terraform_plugin_test.tf line 15, in resource "google_dataproc_metastore_service" "test_resource":
          15:     autoscaling_config {
        Blocks of type "autoscaling_config" are not expected here.
--- FAIL: TestAccDataprocMetastoreService_dataprocMetastoreServiceAutoscalingMaxScalingFactorExample (0.68s)
FAIL
```

### Nightly build test history

https://hashicorp.teamcity.com/test/1113451946542864437?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS&expandTestHistoryChartSection=true&expandedTest=build%3A%28id%3A254016%29%2Cid%3A2000000015
https://hashicorp.teamcity.com/test/5906312384789268776?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS&expandTestHistoryChartSection=true&expandedTest=build%3A%28id%3A254016%29%2Cid%3A2000000016
https://hashicorp.teamcity.com/test/6778169763609926231?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS&expandTestHistoryChartSection=true&expandedTest=build%3A%28id%3A251323%29%2Cid%3A2000000017

b/375372462

**Comment by SarahFrench** [comment]
This is a case of an acceptance test present in the GA provider being made to contain a beta-only field

**Comment by wyardley** [comment]
Seems like this was fixed in #12106

**Comment by melinath** [comment]
yep, thanks!
